aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -599,7 +609,6 @@ public static ModifyDbInstanceRequest modifyDbInstanceAfterCreateRequest(final R
                 .engineVersion(model.getEngineVersion())
```

Comment:
From: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/oracle-cdb-converting.html
> When you upgrade your database engine version, you can't change the database architecture in the same operation. Therefore, to upgrade an Oracle Database 19c non-CDB to an Oracle Database 21c CDB, you first need to convert the non-CDB to a CDB in one step, and then upgrade the 19c CDB to a 21c CDB in a separate step.

I think if we try to do the CDB conversion and engine version upgrade from 19c to 21c at the same time, it's gonna be problematic on CFN handler. Can you please confirm if we're good to go with this scenario for both restoring and updating?